s3-lambda-ses-sam/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Send email with pre-signed URL whenever a file is uploaded into S3 bucket",
```

Comment:
Can the title be shortened?

Reply:
I have shorted the title further. Please let me know if any further changes required.